### DISCUSSION TITLE
Console error: Error: Could not load img

### DISCUSSION THREAD

**Discussion by yevhendevbox** [vuejs/test-utils#2201] [Q&A]
Hey, i am currently learning how unit testing works, have some issues. Tests sims like running fine, but in the console got an error like this in every test file where image exist in component:

```
Error: Could not load img: "http://localhost:3000/src/06-shared/assets/images/useful/svg/logo-sm.svg?inline"
    at onErrorWrapped (/home/yevhen/work/app/node_modules/jsdom/lib/jsdom/browser/resources/per-document-resource-loader.js:38:19)
    at Object.check (/home/yevhen/work/app/node_modules/jsdom/lib/jsdom/browser/resources/resource-queue.js:72:23)
    at /home/yevhen/work/app/node_modules/jsdom/lib/jsdom/browser/resources/resource-queue.js:124:14
    at processTicksAndRejections (node:internal/process/task_queues:95:5) Error: connect ECONNREFUSED 127.0.0.1:3000
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1487:16) {
  errno: -111,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 3000
}
```
I don't understand what is trigger this error, also made an fresh vue app with same spec, run test in there and there's no such an issues some how. I also check vitest.config.ts and compare it to fresh project. Sims like every thing is same basically.. Any tips will be highly appreciate.

**Comment by cexbrayat** [comment]
Hi @yevhendevbox 

That's really hard to say without more context. Maybe you can check if all your dependencies are up to date as a first step.

If that doesn't help, then try to provide us a small repro online using https://stackblitz.com/github/vuejs/create-vue-templates/tree/main/typescript-vitest?file=src%2Fcomponents%2F__tests__%2FHelloWorld.spec.ts and we'll take a look

**Comment by yevhendevbox** [comment]
After i reinstall some dependencies, problem was solved.
